sdk/azurestackhci/Microsoft.Azure.Management.AzureStackHCI/src/Generated/ClustersOperations.cs

```diff
@@ -834,7 +834,7 @@ public async Task<AzureOperationResponse<Cluster>> CreateWithHttpMessagesAsync(s
         /// <return>
         /// A response object containing the response body and response headers.
         /// </return>
-        public async Task<AzureOperationResponse<Cluster>> UpdateWithHttpMessagesAsync(string resourceGroupName, string clusterName, IDictionary<string, string> tags = default(IDictionary<string, string>), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<AzureOperationResponse<Cluster>> UpdateWithHttpMessagesAsync(string resourceGroupName, string clusterName, ClusterPatch cluster, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
This is a breaking change from the previous version.  If Tags can still be used to create a ClusterPatch, then this can be mitigated by adding a partial class that implements the old method in customizations.  This would mean adding the same overload to 'IClustersOperations, ClustersOperations, and ClustersOperationsExtensions'.  Adding a method with the signature:

``` c#
 public async Task<AzureOperationResponse<Cluster>> UpdateWithHttpMessagesAsync(string resourceGroupName, string clusterName, IDictionary<string, string> tags, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

That simply created a ClusterPatch object from the tags and called the existing method would work.